src/msw/radiobox.cpp

```diff
@@ -594,12 +578,11 @@ wxSize wxRadioBox::GetTotalButtonSize(const wxSize& sizeBtn) const
 
     // Add extra space under the label, if it exists.
     if (!wxControl::GetLabel().empty())
-        height += cy1/2;
+        height += wxRendererNative::Get().GetCheckBoxSize((wxWindow*)this).x / 2;
```

Comment:
This should be `.y`, not `.x`, shouldn't it?

Also, probably better to have

     const wxSize radioSize = wxRendererNative::Get().GetCheckBoxSize(const_cast<wxWindow*>(this));

finally, instead of calling it twice.

Reply:
It is only called once per function.